libcontainer/nsenter/nsexec.c

```diff
@@ -904,7 +570,7 @@ void nsexec(void)
 				/* ... wait for mapping ... */
 				write_log(DEBUG, "request stage-0 to map user namespace");
 				if (read(syncfd, &s, sizeof(s)) != sizeof(s))
-					bail("failed to sync with parent: read(SYNC_USERMAP_ACK)");
+					bail("failed to sync with parent: read(SYNC_USERMAP_ACK) got %d", s);
```

Comment:
this is a "short read" when we did not read as many bytes as expected, so showing the value is meaningless (as it's not read).